gait-control/circular-gait.py

```diff
@@ -1,62 +1,29 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jun 29 11:20:09 2017
+
+@author: jrs
+"""
+
 import serial # python -m pip install pyserial
 import numpy as np
-from time import sleep
 import matplotlib.pyplot as plt
+from PneumaticBoard import *
+
+board = PneumaticBoard()
 
-# TODO: determine actual mapping function from curvature -> pressure -> pwm dutycycle
+max_dutycycle = 24 #24 for extensible
 
-if __name__ == "__main__":	
-	# TODO: change these?
-	max_dutycycle = 24
-	
-	tlist = np.arange(0, 20, 0.2)
+tlist = np.arange(0, 10, 0.2)
 	# curvatures vary circularly together
-	k1 = np.cos(tlist)
-	k2 = np.sin(tlist)
+k1 = np.cos(tlist)
+k2 = np.sin(tlist)
 	# note that the two lines below are temporary - we need an actual function to map curvature -> pwm rate
 	# each pwmpair will be used to inflate one chamber or the other on a single actuator 
 	# (negative value => chamber 1, positive value => chamber 2)
-	pwmpair1 = max_dutycycle * k1
-	pwmpair2 = max_dutycycle * k2
-	
-	# show gait shape cycle that we are about to send
-	fig, ax = plt.subplots()
```

Comment:
Having plot of expected pwm trends is nice for debugging! Let's add that to current version.